spring-cloud-starter-dataflow-server-cloudfoundry/pom.xml

```diff
@@ -42,5 +37,17 @@
 			<artifactId>reactor-core</artifactId>
 			<version>${reactor.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>io.pivotal.spring.cloud</groupId>
+			<artifactId>spring-cloud-services-starter-config-client</artifactId>
+			<version>1.1.0.RELEASE</version>
```

Comment:
If `spring-cloud-services-starter-config-client` and `spring-cloud-sso-connnector` are part of a specific release train, can we make a version property for it and use here?